resources/views/queues/summary.html

```diff
@@ -0,0 +1,28 @@
+<div class="yii-debug-toolbar__block">
+    <a href="#" title="" data-key="queue.link">Queues
+        <span class="yii-debug-toolbar__label" data-key="queue.messages">0</span>
+        <span class="yii-debug-toolbar__label" data-key="queue.pushes">0</span>
+        <span class="yii-debug-toolbar__label" data-key="queue.statuses">0</span>
```

Comment:
Could use `title` here to display hints about what these numers mean. Also coloring it differently would be cool.